packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,181 @@
+import { ThreeDotSaving } from "@comet/admin-icons";
+import { Button as MuiButton, ButtonClassKey, ButtonProps, ComponentsOverrides } from "@mui/material";
+import { styled, Theme, useThemeProps } from "@mui/material/styles";
+import * as React from "react";
+
+import { Tooltip, TooltipClassKey } from "../../Tooltip";
+
+export type FeedbackButtonClassKey = "idle" | "loading" | "success" | "fail" | ButtonClassKey;
+
+type OwnerState = Pick<FeedbackButtonProps, "variant" | "color"> & { displayState?: FeedbackButtonDisplayState };
+
+const Button = styled(MuiButton, {
```

Comment:
Please use `createComponentSlot()` instead of `styled()`. 
And maybe name the variable `Root` to match it's slot/styles.